Sources/TuistCore/Utils/Printer.swift

```diff
@@ -34,108 +23,81 @@ public class Printer: Printing {
     // MARK: - Public
 
     public func print(_ text: String) {
-        print(text, output: .standardOputput)
+        printStandardOutputLine(text)
     }
 
-    public func print(_ text: String, output: PrinterOutput) {
-        let writer: InteractiveWriter!
-        if output == .standardOputput {
-            writer = .stdout
+    public func print(error: Error) {
+        if Environment.shared.shouldOutputBeColoured {
+            printStandardErrorLine(error.localizedDescription.red().bold())
         } else {
-            writer = .stderr
+            printStandardErrorLine("Error: \(error.localizedDescription)")
```

Comment:
I am probably a bit old fashion 😅 but I can see some benefits from having "Error:" prefix even the text is colored. When using is part of a bigger chain of tools, we can just search "Error" to find the problem quicker.